src/pack-install.js

```diff
@@ -33,7 +33,9 @@ const packInstall = function(pkgdir) {
         if (pack.status !== 0) {
           return Promise.reject(`npm pack failed:\n${pack.out}`);
         }
-        const tarball = JSON.parse(pack.stdout)[0].filename;
+        const jsonRegex = /\[(.|\n)*\]$/;
+        const jsonString = (pack.stdout).match(jsonRegex)[0];
```

Comment:
Seems like we use the same logic twice, we can move it to a function:

something like:
```js
const jsonRegex = /\[(.|\n)*\]$/;

getParsedDataFromOutput = (stdout) => {
  const jsonString = (result.stdout).match(jsonRegex)[0];

  try {
    return JSON.parse(jsonString);
  } catch (e) {
    return null;
  }
}
```